databricks/koalas/series.py

```diff
@@ -5888,7 +5888,7 @@ def unpack_scalar(sdf):
     return l2[0]
 
 
-def first_series(df):
+def first_series(df) -> "Series":
```

Comment:
Actually it's `Union["Series", pd.Series]`.

Reply:
I guess we can just remove pandas DataFrame support here.

Reply:
I see. Let me change it to `Union["Series", pd.Series]` first.